test/simulator/misc/statuses.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const assert = require('./../../assert');
-const common = require('./../../common');
+const assert = require('../../assert');
+const common = require('../../common');
```

Comment:
One advantage of `./../` is that you can grep the codebase for `require('./` and get all the relative requires. Although I guess you could do it with `require('.` either way... But it feels like it flows better logically.

I definitely don't have a strong opinion here, so I'll let @Slayer95 decide. Personally, I think he's right that if you're going to change this, you might as well change this in a separate pullreq.